src/UserService.Validation/User/CreateUserRequestValidator.cs

```diff
@@ -2,50 +2,57 @@
 using LT.DigitalOffice.UserService.Models.Dto;
 using LT.DigitalOffice.UserService.Validation.User.Interfaces;
 using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace LT.DigitalOffice.UserService.Validation.User
 {
     public class CreateUserRequestValidator : AbstractValidator<CreateUserRequest>, ICreateUserRequestValidator
     {
+        private static Regex NameRegex = new(@"\d");
```

Comment:
Mb better use “/D” and on 16 line for example don’t use negative operator ‘!’
It seems like double negative form

Reply:
it is did not work((